### PR TITLE
Add TIC-80

### PR DESCRIPTION
**Pull request by nesbox**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/464)

### CODE REVIEW

**Comment by nesbox** [comment]
Hi, @KevinLaMS 
WDNotify@microsoft.com confirmed the false positive, but I can't remove the label "Needs: author feedback", possibly I don't have the rights to do it.

**Comment by superusercode** [comment]
@nesbox the CI will remove the author feedback and smartscreen tags if you add or remove a blank line from the bottom of the manifest.

File: manifests/Nesbox/TIC-80/0.70.6.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Description: TIC-80 is a fantasy computer for making, playing and sharing tiny games.
+Homepage: https://tic80.com/
+Installers:
+  - Arch: x86
+    Url: https://github.com/nesbox/TIC-80/releases/download/v0.70.6/tic80_0.70.6.exe
+    Sha256: 921AE257A6187E52862194FA5AEA1EFFF9341690B0CEC92AB9015F71D50CDBBC
+    InstallerType: nullsoft
+    Switches: 
+      Silent: /S
+      SilentWithProgress: /S
```
Comment (superusercode) [inline manifests/Nesbox/TIC-80/0.70.6.yaml:18]:
```suggestion
    InstallerType: nullsoft
```

those switches are already included in the nullsoft type

**Comment by KevinLaMS** [comment]
When running the app, it simply shuts down.  I am not sure why it does not work.  Maybe it is missing a dependency?

**Pull request closed without merging**